.travis.yml

```diff
@@ -1,14 +1,18 @@
 # This file was generated automatically from conda-smithy. To update this configuration,
-# update the conda-forge.yaml and/or the recipe/meta.yaml.
+# update the conda-forge.yml and/or the recipe/meta.yaml.
 
-language: objective-c
+language: generic
+
+os: osx
+osx_image: beta-xcode6.1
 
 env:
   matrix:
     
     - CONDA_PY=27
     - CONDA_PY=34
     - CONDA_PY=35
+    - CONDA_PY=36
```

Comment:
As noted it seems that the upload script is leaking things back into the `main` label even when a different label (e.g. `prerelease` or `broken`) is set or specified. This includes packages that were uploaded previously and are not being uploaded from an active CI build. Just suffered from this again with `python` today. Pretty painful trying to fix everything the way it should be.